Utilities/Utilities.csproj

```diff
@@ -16,6 +16,10 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="coverlet.collector" Version="6.0.0">
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+      <PrivateAssets>all</PrivateAssets>
+    </PackageReference>
```

Comment:
I think it should be only added to Test projects (see https://github.com/coverlet-coverage/coverlet?tab=readme-ov-file#installation-coverletcollector)

Reply:
Understood. Will Create new PR with updated Test projects. Thanks!

Reply:
You can use this PR, just push the changes to the same branch and they'll appear here

Reply:
Will do. Sorry for the inconvenience. Thanks!